src/Nethermind/Nethermind.Blockchain/BlockTreeExtensions.cs

```diff
@@ -10,5 +10,14 @@ public static class BlockTreeExtensions
         public static ReadOnlyBlockTree AsReadOnly(this IBlockTree blockTree) => new(blockTree);
 
         public static BlockHeader? GetProducedBlockParent(this IBlockTree blockTree, BlockHeader? parentHeader) => parentHeader ?? blockTree.Head?.Header;
+
+        public static (bool isSyncing, long headNumberOrZero, long bestSuggested) IsSyncing(this IBlockTree blockTree, int maxDistanceForSynced = 0)
```

Comment:
I think we can rename it for simplicity. Normally our head should be on Genesis by default, if it isn't there is something wrong. Or we can just make it nullable here.

```suggestion
        public static (bool isSyncing, long headNumber, long bestSuggested) IsSyncing(this IBlockTree blockTree, int maxDistanceForSynced = 0)
```